users/utils.py

```diff
@@ -0,0 +1,26 @@
+import jwt
+
+from django.http import JsonResponse
+
+from BeerBnB.settings import SECRET_KEY
+from .models          import User
+
+
+def login_decorator(func):
+	def wrapper(self, request, *args, **kwargs):
+		try:
+			token   = request.headers.get("Authorization")
+			payload = jwt.decode(token, SECRET_KEY, algorithms = "HS256")
+
+			if not User.objects.filter(id = payload.get("id")).exists() :
+				return JsonResponse({"message" : "INVALID_MEMBER"}, status = 400)
+
+			user         = User.objects.get(id = payload["id"])
+			request.user = user
+
+		except jwt.exceptions.DecodeError:
+			return JsonResponse({"message" : "INVALID_TOKEN"}, status = 400)
+
+		return func(self, request, *args, **kwargs)
+
+	return wrapper
```

Comment:
인덴트 4칸으로 맞춰주세요. 기존에 merge된 모든 코드들이 indent 4로 맞춰져 있기 때문에, 팀 양식에 맞춰주세요~

```suggestion
def login_decorator(func):
    def wrapper(self, request, *args, **kwargs):
        try:
	    token   = request.headers.get("Authorization")
	    payload = jwt.decode(token, SECRET_KEY, algorithms = "HS256")

	    if not User.objects.filter(id = payload.get("id")).exists() :
	        return JsonResponse({"message" : "INVALID_MEMBER"}, status = 404)

            user = User.objects.get(id = payload["id"])
	    request.user = user

        except jwt.exceptions.DecodeError:
	    return JsonResponse({"message" : "INVALID_TOKEN"}, status = 403)

	return func(self, request, *args, **kwargs)

    return wrapper
```

Reply:
추가로 데코레이터에서 `positional arguments`와 `keyword args`를 사용하시는 이유가 있으신가요?

Reply:
status 코드도 정확하게 반환해주세요!